L1Trigger/L1TCaloLayer1/plugins/L1TCaloLayer1.cc

```diff
@@ -81,9 +81,13 @@ class L1TCaloLayer1 : public edm::EDProducer {
   edm::EDGetTokenT<HcalTrigPrimDigiCollection> hcalTPSource;
   std::string hcalTPSourceLabel;
   
-  std::vector< std::vector< std::vector < uint32_t > > > ecalLUT;
-  std::vector< std::vector< std::vector < uint32_t > > > hcalLUT;
-  std::vector< std::vector< uint32_t > > hfLUT;
+  std::vector< std::vector< std::vector< std::vector < uint32_t > > > > ecalLUT;
```

Comment:
Hi @rekovic - i had problems with 3 layers of vectors, now you have four:(

so what are these things? it seems some have well understood (eg, hardwired) sizes.